AdAway/src/main/res/values-de/strings.xml

```diff
@@ -184,9 +182,9 @@
     </string-array>
 
     <!-- Scan Adware -->
-    <string name="scan_adware_description">Hier können Sie nach Adware suchen, die nicht von AdAway blockiert werden kann. Diese Anwendungen belästigen den Nutzer beispielsweise durch Airpush-Benachrichtigungen, die auch, wenn die Anwendung gerade nicht genutzt wird, in der Benachrichtungsleiste angezeigt werden. Gefundene Anwendungen können mit einem Klick auf den Eintrag deinstalliert werden!</string>
+    <string name="scan_adware_description">Hier können Sie nach bösen Adware-Anwendungen suchen, die nicht von AdAway geblockt werden können. Diese Apps verwenden beispielsweise Airpush-Benachrichtigungen, die selbst dann angezeigt werden, wenn die App nicht ausgeführt wird, oder sogar den Klingelton ändern. Die einzige verfügbare Gegenmaßnahme besteht darin, diese Apps zu deinstallieren, indem Sie auf die Listenelemente klicken!</string>
```

Comment:
`bösartiger Adware` is the phrase to use here. The word adware already implies that it is an application.